docs/TempUG.md

```diff
@@ -103,15 +103,1078 @@ This project is based on the [AddressBook-Level3 project](https://se-education.o
+```
+Failure outputs:
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+<br>
+
+### Deleting a task : `deleteTask`
+
+About the feature
+
+Format: `command COMPULSORY [optional]`
+* Format info 1
+* Format info 2
+
+<box type="tip" seamless>
+
+**Tips:**
+- Tip 1
+- Tip 2
+
+</box>
+
+Example usages:
+* `some code here`
+* `another code here`
+
+Success outputs:
+* Input: `code with compulsory parameters`
+* Input: `code with compulsory and optional parameters`
+```
+This block of code is for success outputs
+```
+Failure outputs:
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+<br>
+
+### Editing a student : `editPerson`
+
+About the feature
+
+Format: `command COMPULSORY [optional]`
+* Format info 1
+* Format info 2
+
+<box type="tip" seamless>
+
+**Tips:**
+- Tip 1
+- Tip 2
+
+</box>
+
+Example usages:
+* `some code here`
+* `another code here`
+
+Success outputs:
+* Input: `code with compulsory parameters`
+* Input: `code with compulsory and optional parameters`
+```
+This block of code is for success outputs
+```
+Failure outputs:
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+<br>
+
+### Editing a lesson : `editLesson`
+
+About the feature
+
+Format: `command COMPULSORY [optional]`
+* Format info 1
+* Format info 2
+
+<box type="tip" seamless>
+
+**Tips:**
+- Tip 1
+- Tip 2
+
+</box>
+
+Example usages:
+* `some code here`
+* `another code here`
+
+Success outputs:
+* Input: `code with compulsory parameters`
+* Input: `code with compulsory and optional parameters`
+```
+This block of code is for success outputs
+```
+Failure outputs:
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+* Input: `invalid command code here`
+* Explanation and solution here, this is because the flag has an incorrect value, bla bla bla
+```
+Invalid command with the error message here
+```
+
+<br>
+
+### Locating students/lessons by name : `find`
```

Comment:
Would "Finding" be better than "Locating"?

Reply:
Agreed